DS-Assignment-3/commandPrompt/Directory.java

```diff
@@ -0,0 +1,51 @@
+package commandPrompt;
+
+import java.util.ArrayList;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Directory contains details of directories in tree
+ * 
+ * @author Arjita
+ *
+ */
+public class Directory {

```

Comment:
Add a method to add sub directory in this class only. So that we don't require to get the list and then add the sub-directory to it from the VCP class